docs/opcodes/37/homestead.mdx

```diff
@@ -0,0 +1,8 @@
+## Gas
```

Comment:
I think the group should be `System operations` here?

Reply:
This is the groups from the yellow paper, they are correct :-)

Also fyi, the groups are set in `docs/opcodes/37.mdx`, not this file.